index.js

```diff
@@ -26,8 +26,14 @@ function validateSlackRequest (slackAppSigningSecret, httpReq, logging) {
   if (!slackAppSigningSecret || typeof slackAppSigningSecret !== 'string' || slackAppSigningSecret === '') {
     throw new Error('Invalid slack app signing secret')
   }
-  const xSlackRequestTimeStamp = httpReq.get('X-Slack-Request-Timestamp')
-  const SlackSignature = httpReq.get('X-Slack-Signature')
+  const get = (
+    httpReq.get
+    ||(function(str) {
+      return this.headers[str.toLowerCase()]
+    })).bind(httpReq); // Fix for #5
+
+  const xSlackRequestTimeStamp = get('X-Slack-Request-Timestamp')
+  const SlackSignature = get('X-Slack-Signature')
   const bodyPayload = fixedEncodeURIComponent(querystring.stringify(httpReq.body).replace(/%20/g, '+')) // Fix for #1
```

Comment:
Fetching and parsing of the body are missing

Reply:
Yes! I suppose I accidentally added the PR to `validate-slack-request` for Next.js, which applies [its own middleware](https://nextjs.org/docs/api-routes/api-middlewares) to the `IncomingMessage` for parsing HTTP request bodies.